app/components/UI/NavbarBrowserTitle/index.js

```diff
@@ -119,7 +119,7 @@ class NavbarBrowserTitle extends PureComponent {
 		return (
 			<TouchableOpacity onPress={this.onTitlePress} style={styles.wrapper}>
 				<View style={styles.currentUrlWrapper}>
-					{icon && <Image style={styles.siteIcon} source={{ uri: icon }} />}
+					{icon ? <Image style={styles.siteIcon} source={{ uri: icon }} /> : null}
```

Comment:
I'd keep the conditional rendering with `&&` but casting a boolean on the variable (we do the same in the rest of the app but with `!!`, using `Boolean` is just a personal preference)

```suggestion
					{Boolean(icon) && <Image style={styles.siteIcon} source={{ uri: icon }} />}
```